src/middleware/verify-and-receive.ts

```diff
@@ -7,15 +7,14 @@ import {
 
 export function verifyAndReceive<T extends All>(
   state: State<T>,
-  event: EventTypesPayload[T] & { signature: string }
+  event: EventTypesPayload[T] & { signature?: string }
```

Comment:
why making signature optional here? I don't think it makes sense to call `.verifyAndReceive()` but not have a signature set?

Reply:
I agree, I was just trying out. This is still a WIP. Will remove this :)